spec/controllers/api/lists_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe Api::ListsController do
+  describe "POST create" do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
+      @new_list = build(:list)
+      post :create, list: { name: @new_list.name, description: @new_list.description }
+    end
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "returns json content type" do
+      expect(response.content_type).to eq 'application/json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.